packages/expo-modules-core/ios/ReactDelegates/ExpoReactDelegate.swift

```diff
@@ -17,7 +17,7 @@ public class ExpoReactDelegate: NSObject {
     initialProperties: [AnyHashable: Any]?,
     launchOptions: [UIApplication.LaunchOptionsKey: Any]?
   ) -> UIView {
-    return self.handlers.lazy
+    return self.handlers
```

Comment:
This should not be lazy because even though some modules will not return a view, `DevMenu` for example, it still needs to be called to set up it's bridge reference